slack-api-client/src/main/java/com/slack/api/methods/request/apps/event/authorizations/AppsEventAuthorizationsListRequest.java

```diff
@@ -15,7 +15,7 @@
 
     /**
      * You'll receive an event_context identifying an event in each event payload sent to your app.
-     *
+     * <p>
```

Comment:
why the `<p>`

Reply:
I ran the code formatter to all the code in the same package and the code formatter does it automatically. A blank line is always treated this way.